core/embed/extmod/modtrezorui/modtrezorui-display.h

```diff
@@ -596,7 +596,6 @@ STATIC const mp_rom_map_elem_t mod_trezorui_Display_locals_dict_table[] = {
      MP_ROM_PTR(&mod_trezorui_Display_clear_save_obj)},
     {MP_ROM_QSTR(MP_QSTR_WIDTH), MP_ROM_INT(DISPLAY_RESX)},
     {MP_ROM_QSTR(MP_QSTR_HEIGHT), MP_ROM_INT(DISPLAY_RESY)},
-    {MP_ROM_QSTR(MP_QSTR_FONT_SIZE), MP_ROM_INT(FONT_SIZE)},
```

Comment:
please also drop the corresponding `/// FONT_SIZE` comment

Reply:
Fixed 21c051c91d0c2deb7eddd57be7f31a3d4920da20.